src/components/ProductLinkGrid/styles.module.scss

```diff
@@ -0,0 +1,14 @@
+.gridContainer {
+  display: grid;
+  gap: 20px;
+  grid-template-columns: repeat(2, 1fr);
+  list-style: none;
+  margin: 0 auto;
+  padding: 40px 20px;
+}
+
+@media (min-width: 768px) {
```

Comment:
This is fine but a shortcut/feature of SCSS is being able to nest media queries under class definitions like:
```
.gridContainer {
  /* mobile styles */

  @media (min-width: 768px) {
    /* desktop styles */
  }
}
```